tasks/main.yml

```diff
@@ -56,16 +59,22 @@
   command: gpg2 --import {{ aptly_gpg_key_dir }}/key.sec
   ignore_errors: yes
 
+# ignore 'already in secret keyring' error
+- name: import sec key to gnupg
+  command: gpg2 --import {{ aptly_gpg_key_dir }}/key.sec
+  ignore_errors: yes
+
 # end key creation tasks
 
-# CLI conflict - https://github.com/smira/aptly/issues/234
+# # CLI conflict - https://github.com/smira/aptly/issues/234
 - name: start aptly API (will conflict with command line until stopped)
-  command: sh -c "nohup aptly api serve 2>&1 >/dev/null &"
+  command: sh -c "nohup aptly api serve -no-lock 2>&1 >/dev/null &"
```

Comment:
what does no-lock do?

Reply:
Enables the CLI & API service to run concurrently by not locking the database to the api process. There's a note that outlines the feature at the top of the page here: https://www.aptly.info/doc/api/